apps/server/src/modules/school/service/school.service.ts

```diff
@@ -52,6 +54,14 @@ export class SchoolService {
 		return response;
 	}
 
+	private enableOauthMigration(schoolDo: SchoolDO) {
+		if (schoolDo.features && !schoolDo.features.includes(SchoolFeatures.OAUTH_PROVISIONING_ENABLED)) {
```

Comment:
Array.isArray(schoolDo.features) && ....

Reply:
This is not necessary since this is TypeScript. It can only be an Array (or undefined).

Reply:
TS do not exist to run time.

Reply:
but it comes from mikroorm, which either successfully loads the entity, or throws an error...